scenarios/21a_Abducted_Bride.cfg

```diff
@@ -303,7 +299,7 @@
         {MSGm_Lorin _"I know."}
         {IF_HAVE_UNIT_ANY Elorain}
         {RECALL Elorain}
-        {MESSAGE (Elorain) (portraits/elorain.png) (Elorain) _"My Lord, this seems to be an ideal task for an Elvish Avenger. I can sneak right up to the cage and free that girl!"}
+        {MESSAGE (Elorain) (portraits/elorain.png) (Elorain) _"My Lord, this seems to be an ideal task for an Elvish Avenger, and, in particular, me. I can sneak right up to the cage and free that girl!"}
```

Comment:
I think the original is better.

Reply:
the point is to have the message be slightly different so players can tell the difference between Elorain and other Elvish Avengers; I got kind of confused about who had been recalled by this event on my most recent playthrough

Reply:
Well then maybe have some phrasing that doesn't involve 3 commas. Like just "... an Elvish Avenger like me". But also it doesn't really have to be him, you could just use another Avenger or even a Ranger. So I don't think it's needed.

Reply:
ok changed in [d9dba4b](https://github.com/nemaara/A_New_Order/pull/43/commits/d9dba4bfd0c5918bc3d493d42d8d5d32782419ea)